packages/esm-ward-app/src/ward-patient-card/row-elements/ward-patient-pending-transfer.tsx

```diff
@@ -25,12 +28,27 @@ const WardPatientPendingTransfer: React.FC<WardPatientTransferProps> = ({ wardPa
     return '';
   }, [dispositionType, dispositionLocation]);
 
+  const launchCancelAdmissionForm = () => {
+    launchWorkspace<WardPatientWorkspaceProps>('cancel-admission-request-workspace', {
+      wardPatient,
+      WardPatientHeader,
+    });
+  };
+
   if (!(dispositionType === 'TRANSFER' || dispositionType === 'DISCHARGE')) return null;
 
   return (
     <div className={styles.wardPatientCardDispositionTypeContainer}>
-      <Movement className={styles.movementIcon} size="24" />
+      <Movement className={styles.movementIcon} size={24} />
       {message}
+      <Button
```

Comment:
nit: It's preferable to use [IconButton](https://react.carbondesignsystem.com/?path=/docs/components-combobox--overview) for the icon-only button variant.